src/network/networkd-link.h

```diff
@@ -201,6 +201,9 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(Link*, link_unref);
 DEFINE_TRIVIAL_DESTRUCTOR(link_netlink_destroy_callback, Link, link_unref);
 
 int link_get(Manager *m, int ifindex, Link **ret);
+int link_get_by_name(const Manager *m, const char *ifname, Link **ret);
+int link_update_by_name(Link *link);
```

Comment:
It seems `link_update_by_name()` can be static.

Reply:
It may be used somewhere later. Let's keep it that way.